.circleci/e2e-job-ignore-patterns.txt

```diff
@@ -6,5 +6,15 @@
 .log
 .pdf
 .gitignore
-.java
-.kt
\ No newline at end of file
+Test.java
+.kt
+.tsx
+.git
+yarn.lock
```

Comment:
Why would we want to ignore this?

Reply:
Missed this comment. I added it here during testing.  I removed it from ignore.